docs/packaging/packaging-with-visual-studio.md

```diff
@@ -0,0 +1,81 @@
+(and optionally check the "C++ WinUI app development tools" box for development in C++). 
+The Windows 11 SDK 10.0.26100.0 (or greater) is also required.
+
+![image](images/vs/01-VisualStudio-Required-Workloads.png)
+
+## Step 2 - Create your app project
+In Visual Studio, create a new C# or C++ project using the template *Blank App, 
+Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)*. 
+Click "Create" and then select "10.0.26100.0" (or later) for Target Platform Version (TPV) 
+and Target Platform Minimum Version (TPMinV). 
+
+![image](images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png)
+![image](images/vs/03-New-Project.png)
+![image](images/vs/04-New-Windows-Project.png)
+
+## Step 3 - Set NuGet to use Microsoft.Windows.SDK.BuildTools version 10.0.26100.1 or later
+Go to Project -> "Manage NuGet Packages" to install Microsoft.Windows.SDK.BuildTools 
+version 10.0.26100.1 (or later).
+
+![image](images/vs/05-Manage-NuGet-Packages.png)
+![image](images/vs/06-Microsoft-Windows-SDK-BuildTools.png)
+![image](images/vs/07-SDK-BuildTools-Info.png)
+
+## Step 4 - Edit the Packaging.appxmanifest file
+In the manifest file, the following changes will need to be made:
+
+**Note**: Isolated win32 applications are not compatible with other application types within the same package.
+
+* Add `xmlns:previewsecurity2="http://schemas.microsoft.com/appx/manifest/preview/windows10/security/2"`
+to the `<Package>` element if it's not there already.
+
+	* Add `previewsecurity2` to `IgnorableNamespaces` at the end of the `<Package>` element.
+
+* Add `xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"` to the `<Package>` element
+if it's not there already.
+
+	* Add `uap10` to `IgnorableNamespaces` at the end of the `<Package>` element.
+
+* In `<Dependencies>` change `TargetDeviceFamily` to
+`<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.25357.0" MaxVersionTested="10.0.25357.0" />`.
+
+	* **Note**: Not all features are available in the minimum build, check out the [release notes](../../relnotes/windows-release-notes.md) for more detailed information.
+
+* In `<Application>` replace any existing entrypoint/trustlevel/runtimebehavior with
+`uap10:TrustLevel="appContainer" previewsecurity2:RuntimeBehavior="appSilo"`.
+
+* In `<Application>` extensions, remove any `EntryPoints=*` or `Executable=*` as those are inherited from the parent `<Application>`
+
+* Add `desktop7:Scope="user"` to the extension element for `windows.protocol`.
+
+* **Note**: By default, Visual Studio will automatically add `<rescap:Capability name="runFullTrust">` to
+`<Capabilities>` due to the app being a packaged Win32. This should be removed unless
+the app has other manifested extensions which can affect the user global state, such as
+`comServer` or `FirewallRules`, since those require the `runFullTrust` capability.
+
+![image](images/vs/09-PackagingAppx-File.png)
```

Comment:
Your image does not quite match the description.

1. There are wavy underlines in the image which indicates that this is probably wrong. That's not a good signal to send. 
2. The `MinVersion` in the image does not match the description.
3. I'd really like the changed (important) fields in the image get highlighted. Similar to the MPT docs.
4. Is this purely based on the example manifest given by the "hello world" example? It would be nice if we can trim off some unrelated and unnecessary stuff like the logos and make this a minimum example, which would be much user friendly.

Reply:
Could you please share the MPT docs you're referring to? Getting highlighted is a little tricky. I could try making a git repo and see if that highlights the changes, but the equivalent image in the doc is not highlighted which I linked earlier.

Reply:
The image I posted does publish and display correctly. Luke mentioned IntelliSense might be a bit behind so that could be why there's waves?

Reply:
Ah sorry I was under the impression that the MPT docs has the highlighting. It's probably in one of our slides. Just do a red rectangle on the fields that need attention. I believe snipping tool/paint can simply do that.
![image](https://github.com/microsoft/win32-app-isolation/assets/125088426/c9cf10a8-85f5-4e5d-94ed-c9589ae86441)

Reply:
Is there a way to temporarily disable the intellisense so we can get a good snapshot?